x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -724,6 +726,13 @@ export class TaskRunner<
             this.logger.debug(message, {
               tags: [this.ruleType.id, ruleId, 'rule-run-failed', errorSourceTag],
             });
+          } else if (isClusterBlockError(err)) {
```

Comment:
It's seems it's the same code block as it was after first condition. (or I just missed something)
Maybe it can be something like this:
```
if (isAlertSavedObjectNotFoundError(err, ruleId) || isClusterBlockError(err)) {
```

Reply:
Initially the message was different, then I used the same template and forgot to merge the if cases...
Fixed it, thanks for catching it.